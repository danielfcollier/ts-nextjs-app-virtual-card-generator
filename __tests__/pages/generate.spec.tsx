import { render } from '@testing-library/react';
import Generate from '@/pages/generate';

it('Snapshot of Generate page unchanged', () => {
  const { container } = render(<Generate />);
  expect(container).toMatchSnapshot();
});
