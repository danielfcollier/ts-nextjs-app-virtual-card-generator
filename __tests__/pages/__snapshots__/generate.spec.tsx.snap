// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of Generate page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <h1
      class="title"
    >
      QR Code Image Generator
    </h1>
    <form
      class=""
      novalidate=""
    >
      <div
        class="mb-3 input-group"
      >
        <span
          class="input-group-text"
        >
          Name
        </span>
        <input
          class="form-control"
          id="name"
          required=""
          value=""
        />
      </div>
      <div
        class="mb-3 input-group"
      >
        <span
          class="input-group-text"
        >
          LinkedIn URL
        </span>
        <input
          class="cards form-control"
          id="linkedin"
          required=""
          value=""
        />
      </div>
      <div
        class="mb-3 input-group"
      >
        <span
          class="input-group-text"
        >
          GitHub URL
        </span>
        <input
          class="cards form-control"
          id="github"
          value=""
        />
      </div>
      <button
        class="button btn btn-outline-dark btn-lg"
        type="submit"
      >
        Generate Image
      </button>
    </form>
  </div>
</div>
`;
